meinberlin/apps/plans/migrations/0006_make_plan_district_required.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.9 on 2018-01-22 10:45
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def delete_plans(apps, schema_editor):
+    plan_model = apps.get_model('meinberlin_plans', 'Plan')
+    for plan in plan_model.objects.all():
+        plan.delete()
```

Comment:
delete may also be called on a queryset like `plan_model.objects.all().delete()` but i would not update this PR. just catched my eye

Reply:
Ah, ok, good to know, thanks!